fibbingnode/southbound/lsdb.py

```diff
@@ -466,7 +491,35 @@ def process_lsa(self):
                                  for part in lsa_info.split(SEP_GROUP) if part]
                     hdr = LSAHeader.parse(lsa_parts.pop(0))
                     lsa = LSA.parse(hdr, lsa_parts)
+                    new_seqnum = hdr.lsa_seqnum
```

Comment:
Directly put the seqnum in the LSA -- this will need a cleaner refactoring eventually but the current PR needs to be finished first.

lsa.seqnum = hdr.lsa_seqnum